zenith_metrics/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+pub use prometheus::{register_histogram, Histogram};
+pub use prometheus::{register_histogram_vec, HistogramVec};
+pub use prometheus::{register_int_counter, IntCounter};
+pub use prometheus::{register_int_counter_vec, IntCounterVec};
+
+use std::io;
+
+pub mod metrics;
+
+pub struct CountedReader<T> {
```

Comment:
Seem like unused code

Reply:
True. I thought somebody might need it later. Here's an example:

```rust
let reader = CountedReader::new(stream, || {
    COUNTER.inc();
});
```

I'll stash it if you insist that we don't need this.

Reply:
I am okay to leave that, but please put the comment with this usage example -- that way it is more clear what is going on. Also let's put that to zenith_utils/src/metrics helper or something like this.